1971.py

```diff
@@ -0,0 +1,48 @@
+# class Solution:
```

Comment:
Manually commenting out any portion of your code in a version controlled environment is not a good practice. I would create a new file for a different implementation, or just overwrite the existing version with the new version. Ultimately, providing you access to older versions of your code is exactly what a version control system is supposed to do.

Reply:
Instead, I listed all the solutions in one file. I don't need it to be compiled.